contracts/HybridCustody.cdc

```diff
@@ -503,27 +519,29 @@ pub contract HybridCustody {
     /// able to manage all ChildAccount resources it shares, without worrying about whether the upstream parent can do
     /// anything to prevent it.
     /// 
-    pub resource ChildAccount: AccountPrivate, AccountPublic, MetadataViews.Resolver {
+    access(all) resource ChildAccount: AccountPrivate, AccountPublic, ViewResolver.Resolver, Burner.Burnable {
+        access(all) event ResourceDestroyed(uuid: UInt64 = self.uuid, address: Address = self.childCap.address, parent: Address = self.parent)
+
         /// A Capability providing access to the underlying child account
-        access(self) let childCap: Capability<&{BorrowableAccount, OwnedAccountPublic, MetadataViews.Resolver}>
+        access(self) let childCap: Capability<&{BorrowableAccount, OwnedAccountPublic, ViewResolver.Resolver}>
 
         /// The CapabilityFactory Manager is a ChildAccount's way of limiting what types can be asked for by its parent
         /// account. The CapabilityFactory returns Capabilities which can be casted to their appropriate types once
         /// obtained, but only if the child account has configured their factory to allow it. For instance, a
         /// ChildAccount might choose to expose NonFungibleToken.Provider, but not FungibleToken.Provider
-        pub var factory: Capability<&CapabilityFactory.Manager{CapabilityFactory.Getter}>
+        access(self) var factory: Capability<&{CapabilityFactory.Getter}>
```

Comment:
Revisiting this after the latest local migration result, I think this will have to be `CapabilityFactory.Manager`. I suppose that makes sense since it's at restrictive of the type as the original field type. The same will have to be the case with `delegator` - I believe that will have to be `Delegator`